api/lib/ZwaveClient.ts

```diff
@@ -4602,9 +4599,11 @@ class ZwaveClient extends TypedEventEmitter<ZwaveClientEventCallbacks> {
 		}
 	}
 
-	private _onInclusionStarted(secure: boolean) {
+	private _onInclusionStarted(strategy: InclusionStrategy) {
+		const secure = strategy !== InclusionStrategy.Insecure
 		const message = `${secure ? 'Secure' : 'Non-secure'} inclusion started`
 		this._updateControllerStatus(message)
+		// FIXME: Should the frontend also accept the strategy instead of the boolean?
 		this.emit('event', EventSource.CONTROLLER, 'inclusion started', secure)
```

Comment:
See FIXME comment

Reply:
Frontend uses `startInclusion` and passes the mode:

https://github.com/zwave-js/zwave-js-ui/blob/80c1cd86edf77c6b375ae2046e7d506d52f483d3/src/components/dialogs/DialogNodesManager.vue#L995-L998


https://github.com/zwave-js/zwave-js-ui/blob/80c1cd86edf77c6b375ae2046e7d506d52f483d3/src/components/dialogs/DialogNodesManager.vue#L1149-L1156

Reply:
I meant the event that is emitted below the comment. Right now it gets passed `secure`, I wonder if it should now include the strategy instead.

Reply:
that event actually is only emittet to MQTT when `Emit events` flag is enabled in MQTT settings, nothing used by UI btw. I can fix that and use the inclusion strategy instead